docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -467,116 +500,55 @@ roleRef:
+Now imagine you've magically become an application developer.  
 
-The only environment variable you will need for this next set of steps is `APPLICATION_DB_NAME`, and you can re-export that as:
+You can no longer access the secrets we stored above in environment variables.
+Open a new terminal so that all those variables are gone.
+
+You know only one thing -- the name of the database:
 
 ```bash
 export APPLICATION_DB_NAME=quick_start_db
 ```
 
-### Build Application Deployment Manifest
-
-In this section, you build the deployment manifest for the application. The manifest includes a section for specifying the pod template (`$.spec.template`), it is here that we will declare the application container and Secretless Broker sidecar container.
-
-Below is the base manifest that you will be building upon:
-_quick-start-application.yml_
-```yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: quick-start-application
-  labels:
-    app: quick-start-application
-spec:
-  replicas: 3
-  selector:
-    matchLabels:
-      app: quick-start-application
-  template:
-    metadata:
-      labels:
-        app: quick-start-application
-    spec:
-      serviceAccountName: quick-start-application
-      automountServiceAccountToken: true
-      containers:
-      # to be completed in the following steps
-```
-
-The manifest should be saved as a file named **quick-start-application.yml**. As you can see above, to start off the base manifest you declare a deployment of 3 replicas with associated metadata, and assign the *quick-start-application* ServiceAccount (that we created earlier) to the pod.
+### Sample Application Overview
 
-The additional steps to complete building the manifest are provided to be informative:
-+ [Add & Configure Application Container](#add--configure-application-container)
-+ [Add & Configure Secretless Broker Sidecar Container](#add--configure-secretless-broker-sidecar-container)
+The application we'll be deploying is a [pet store demo
+application](https://github.com/conjurdemos/pet-store-demo) with a simple API:
 
-A [Completed Application Deployment Manifest](#completed-application-deployment-manifest) is provided in the last section, where you'll actually create the **quick-start-application.yml**.
+- `GET /pets` lists all the pets
+- `POST /pet` adds a pet
 
-#### Add & Configure Application Container
+Its PostgreSQL backend is configured using a `DB_URL` environment variable: 
 
-The Secretless Broker sidecar container has a shared network with the application container. This allows us to point the application to `localhost` where the Secretless Broker is listening on port `5432`, in accordance with the configuration stored in ConfigMap.
+```
+postgresql://localhost:5432/${APPLICATION_DB_NAME}?sslmode=disable
```

Comment:
adding this code block gives it a copy/paste button - it would probably look better in a `<pre>` block